NFT/hardhat.config.js

```diff
@@ -30,5 +30,8 @@ module.exports = {
       },
       chainId: 595,
     },
+  },  
+  mocha: {
+    timeout: 300000
```

Comment:
We already have timeouts specified explicitly in tests. We can migrate to this way of writing them , but the READMEs would need to be changed and we would loose the experience when pointing out which example is too complex to run with the default timeout.

I think we should remove these for now and keep in in a TODO if we decide for it. 🤔

Reply:
sure, I think timeout is not necessary, we can just remove these to see if we got any timeout